articles/native-platforms/ios-reactnative.md

```diff
@@ -199,7 +220,7 @@ RCT_EXPORT_METHOD(showTouchID:(NSDictionary *)options callback:(RCTResponseSende
 @end
 ```
 
-> You can also download [LockReactModule.h](https://raw.githubusercontent.com/auth0/native-mobile-samples/master/iOS/basic-sample-reactnative/iOS/Modules/LockReactModule.h) and [LockReactModule.m](https://raw.githubusercontent.com/auth0/native-mobile-samples/master/iOS/basic-sample-reactnative/iOS/Modules/LockReactModule.m) and add them to your Xcode project.
+> You can also download [LockReactModule.h](https://raw.githubusercontent.com/auth0/native-mobile-samples/master/iOS/basic-sample-reactnative/iOS/LockReactModule.h) and [LockReactModule.m](https://raw.githubusercontent.com/auth0/native-mobile-samples/master/iOS/basic-sample-reactnative/iOS/LockReactModule.m) and add them to your Xcode project.
```

Comment:
I updated the react native sample: https://github.com/auth0/native-mobile-samples/pull/1
After pull request is merged link must become working